assets/src/application/ui/layout/espressoTable/ResponsiveTable.tsx

```diff
@@ -39,7 +29,7 @@ const ResponsiveTable = ({
 	onDragUpdate,
 	onDragEnd,
 	...props
-}) => {
+}: any) => {
```

Comment:
Quick "fix" for the error I was getting:
![image](https://user-images.githubusercontent.com/1477453/75282888-74c9d600-581a-11ea-8c48-37b03a823d79.png)